### PR TITLE
Document new select field options

### PR DESCRIPTION
**Pull request by blackbaud-johnly**
Addresses https://github.com/blackbaud/skyux2-docs/issues/865 and https://github.com/blackbaud/skyux2-docs/issues/868.

### CODE REVIEW

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://developer.blackbaud.com/skyux?_sp.spa=pr-select-field-search-031620](https://developer.blackbaud.com/skyux?_sp.spa=pr-select-field-search-031620)

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://developer.blackbaud.com/skyux?_sp.spa=pr-select-field-search-031620](https://developer.blackbaud.com/skyux?_sp.spa=pr-select-field-search-031620)

File: src/app/components/select-field/index.html [hunk 1]
```diff
@@ -40,63 +40,70 @@
...
-    Specifies the selection mode that determines whether users can select one item or multiple items in the select field component's modal. The valid options are <sky-code>single</sky-code>, which displays a text input and allows users to select one item, and <sky-code>multiple</sky-code>, which displays a button and allows users to select multiple items.
+      Specifies the selection mode that determines whether users can select one item or multiple items. The valid options are <sky-code>single</sky-code>, which displays a text input, and <sky-code>multiple</sky-code>, which displays a button. This property accepts <sky-code>string</sky-code> values.
     </sky-demo-page-property>
     <sky-demo-page-property
       propertyName="showAddNewRecordButton"
       isOptional="true"
       defaultValue="false"
     >
-    Indicates whether to display a button for users to add new items directly from the select field modal. You must provide the logic for adding items.
+      Indicates whether to display a button in the picker for users to add items. Consumers must provide the logic to add items. This property accepts <sky-code>boolean</sky-code> values.
```
Comment (Blackbaud-AlexKingman) [inline src/app/components/select-field/index.html]: In the interest of further improvement, should we also mention that consumers would need to tie into the `addNewRecordButtonClick` event to run their custom logic?

Comment (blackbaud-johnly) [reply src/app/components/select-field/index.html]: Good call. Making an update now and will have it in place shortly...

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://developer.blackbaud.com/skyux?_sp.spa=pr-select-field-search-031620](https://developer.blackbaud.com/skyux?_sp.spa=pr-select-field-search-031620)

**Pull request merged**